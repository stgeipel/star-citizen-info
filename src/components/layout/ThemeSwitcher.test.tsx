import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ThemeSwitch from './ThemeSwitcher';

describe('ThemeSwitch', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<ThemeSwitch></ThemeSwitch>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
