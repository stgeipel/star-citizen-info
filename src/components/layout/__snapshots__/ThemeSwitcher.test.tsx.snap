// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemeSwitch matches snapshot 1`] = `
<ReactSwitch
  activeBoxShadow="0 0 2px 3px #3bf"
  boxShadow={null}
  checked={false}
  checkedIcon={
    <FiMoon
      className="inline-block mx-2 my-1"
    />
  }
  className="text-gray-900 dark:text-gray-100 border border-gray-400 dark:border-0"
  disabled={false}
  height={28}
  offColor="#f7fafc"
  offHandleColor="#1a202c"
  onChange={[Function]}
  onColor="#1a202c"
  onHandleColor="#f7fafc"
  uncheckedIcon={
    <FiSun
      className="inline-block mx-2 my-1"
    />
  }
  width={56}
/>
`;
